pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -169,7 +169,7 @@ func TestUpdateStatus(t *testing.T) {
 	storage, statusStorage, server := newStorage(t)
 	defer server.Terminate(t)
 	defer storage.Store.DestroyFunc()
-	ctx := genericapirequest.NewContext()
+	ctx := genericapirequest.NewDefaultContext()
```

Comment:
this is not correct... persistent volumes are not namespaced... this function sets namespace=`default`

Reply:
should be `genericapirequest.WithNamespace(genericapirequest.NewContext(), metav1.NamespaceNone)`